### PR TITLE
rewrite testing documentation

### PR DESCRIPTION
**Pull request by saupchurch**
Restructures the testing documentation to point to the compliance suite and serve as a repository for test configuration files

### CODE REVIEW

File: testing/crg_config.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    
+    studies:
+      - id: E-MTAB-5200-ST0
+        filters:
+          name: Pancancer Analysis of Whole Genomes
+    
+    expressions:
+      - id: 8beada7b93d5e55aa557138b39c6f930
+        filters:
+          tags: cancer
```
Comment (jb-adams) [inline testing/crg_config.yaml]: `format` is now a required filter for each expression, as format must be specified in the `/expressions/search` endpoint. The expression object definition should look like this:

```
expressions:
  - id: 8beada7b93d5e55aa557138b39c6f930
    filters:
    tags: cancer
    format: tsv
```

File: testing/felcat_config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    studies:
+      - id: 6cccbbd76b9c4837bd7342dd616d0fec
+        filters:
+          version: 1.0
+          name: PCAWG
+    
+    expressions:
+      - id: 2a7ab5533ef941eaa59edbfe887b58c4
+        filters:
+          studyID: 6cccbbd76b9c4837bd7342dd616d0fec
```
Comment (jb-adams) [inline testing/felcat_config.yaml]: format is now a required filter for each expression, as format must be specified in the /expressions/search endpoint. The expression object definition should look like this:

```
expressions:
  - id: 2a7ab5533ef941eaa59edbfe887b58c4
    filters:
    studyID: 6cccbbd76b9c4837bd7342dd616d0fec
    format: loom
```

File: testing/master_server_config.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    studies:
+      - id: 6cccbbd76b9c4837bd7342dd616d0fec
+        filters:
+          version: 1.0
+          name: PCAWG
+    
+    expressions:
+      - id: 2a7ab5533ef941eaa59edbfe887b58c4
+        filters:
+          studyID: 6cccbbd76b9c4837bd7342dd616d0fec
```
Comment (jb-adams) [inline testing/master_server_config.yaml]: please add `format: loom` to the Caltech expression filters

File: testing/master_server_config.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    
+    studies:
+      - id: E-MTAB-5200-ST0
+        filters:
+          name: Pancancer Analysis of Whole Genomes
+    
+    expressions:
+      - id: 8beada7b93d5e55aa557138b39c6f930
+        filters:
+          tags: cancer
```
Comment (jb-adams) [inline testing/master_server_config.yaml]: please add `format: tsv` to the CRG expression filters

File: testing/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+## rnaget API Compliance Testing
+
+Implementors can check if their rnaget implementations conform to the specification by using our [compliance suite](https://github.com/ga4gh-rnaseq/rnaget-compliance-suite).
+
+Several sources have written test servers and clients for the rnaget API.  These are available for testing.  Testing configuration files required for the compliance tests can be found in this repository.  A combined configuration file for testing all available servers is also provided.
+
+Implementors who wish to be included in this list can do so by providing their configuration file.
+
+### Compliance report
```
Comment (jb-adams) [inline testing/README.md]: The rnaget-compliance application will actually start the test server automatically if the `--serve` option is specified. Should we mention that? The commands below only need to be run if the server is stopped, and one wants to restart the server with the same archive of web files.

**Comment by saupchurch** [comment]
Changes made

**Pull request merged**